docs/introduction.md

```diff
@@ -0,0 +1,155 @@
+ * provision.yaml
+   * Playbook called to handle installing application to the cluster
+ * deprovision.yaml
+   * Playbook called to handle uninstalling
+ * bind.yaml
+   * Playbook to grant access to another service to use this service, i.e. generates credentials
+ * unbind.yaml
+   * Playbook to revoke access to this service
+ * apb.yaml
+   * Metadata file, exposes parameters the application accepts
+
+The required named playbooks correspond to methods defined by the Open Service Broker API. For example, when the
+Ansible Service Broker needs to provision an APB it will execute the provision.yaml.
+ 
+After the required named playbooks have been generated the files can be used directly to test management of the
+application. A developer may want to work with this directory of files, make tweaks, run, repeat until they are
+happy with the behavior. They can test the playbooks by invoking Ansible directly with the playbook and any
+required variables.
+
+### Ansible Playbook Bundle: Package
+
+The packaging step is responsible for building a container image from the named playbooks for distribution.
+Packaging combines a base image containing an Ansible runtime with ansible artifacts and any dependencies required
+to run the playbooks. The result is a container image with an ENTRYPOINT set to take in several arguments, one of
+which is the method to execute, such as provision, deprovision, etc.
+
+![Package](apb-package.png)
+
+### Ansible Playbook Bundle: Deploy
+
+Deploying an APB means invoking the container and passing in the name of the playbook to execute along with any
+required variables. It’s possible to invoke the APB directly without going through the Ansible Service Broker.
+Each APB is packaged so it’s ENTRYPOINT will invoke ansible when run. The container is intended to be short-lived,
+coming up to execute the ansible playbook for managing the application then exiting.
+ 
+In a typical APB deploy, the APB container will provision an application by running the provision.yaml playbook which
+executes a deployment role. The deployment role is responsible for creating the OpenShift resources, perhaps through
+calling oc create commands or leveraging ansible modules. The end result is that the APB runs Ansible to talk to
+OpenShift to orchestrate the provisioning of the intended application.
+
+![Deploy](apb-deploy.png)
+
+### Summary
+
+The approach discussed has focused on the methods defined by the Open Service Broker API, yet we are envisioning
+additional methods to be added in the future to assist with cases of upgrading, downgrading, testing, verifying, etc.
+
+We are targeting a working demonstration for Red Hat Summit with a deliverable of the Ansible Service Broker in a summer release.
+
+To keep up with developments or to learn more:
+ * [Trello](https://trello.com/b/50JhiC5v/ansible-apps)
+ * Github
+   * [Ansible Playbook Bundle Design Documents](https://github.com/fusor/ansibleplaybookbundle/tree/master/docs)
```

Comment:
Change url to one that works: https://github.com/fusor/ansible-playbook-bundle/tree/master/docs